sdk/tests/client/mod.rs

```diff
@@ -260,8 +260,6 @@ fn is_remainder_or_return(output: &Output, amount: u64, address: Address, native
             return false;
         }
 
-        // assert_eq!(output.as_basic().native_tokens().len(), 0);
```

Comment:
Why were we doing this check?

Reply:
I mean we weren't really doing it but wondering what that could have been

Reply:
It was already commented in the commit that added it at this place :trollface: hhttps://github.com/iotaledger/iota.rs/pull/1356/commits/8f56de6827a2f5f30cd791081cc010c6379d78b6#diff-a98d1b9bdfeb4e19971e5dde8750aab6db35b11bda09b55f052f658ea8822bce
Looks like you moved the validation from the individual tests and didn't completely remove it since it can't work for all the tests

Reply:
Thanks